src/hooks/transactions/useSignTransactions.tsx

```diff
@@ -162,13 +178,18 @@ export const useSignTransactions = () => {
     } catch (error) {
       const errorMessage =
         (error as Error)?.message || (error as string) || ERROR_SIGNING_TX;
+      console.error(errorMessage);
+
       dispatch(
         moveTransactionsToSignedState({
           sessionId,
           status: TransactionBatchStatusesEnum.cancelled
         })
       );
-      onCancel(errorMessage, sessionId);
+      onCancel(
+        errorMessage.includes('cancel') ? TRANSACTION_CANCELLED : errorMessage,
```

Comment:
This string should probably be in an enum

Reply:
Unfortunately, erdjs throws inconsistent errors from different providers...it's hard to sync this logic with all errors thrown